ouroboros-consensus-cardano-tools/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -127,8 +128,8 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
 
     (conwayGenesis, _conwayGenesis) <-
       firstExceptT CardanoProtocolInstantiationConwayGenesisReadError $
-        error "Conway.readGenesis" npcConwayGenesisFile
```

Comment:
TODO This seems like it's in the wrong commit -- it should be in the first commit instead.

Reply:
I think the squash resolved this.